src/util/initialState.js

```diff
@@ -42,7 +42,7 @@ export const initialState = () => {
         contexts: []
       },
     },
-    recentlyEdited: [],
+    recentlyEdited: { [EM_TOKEN]: {} },
```

Comment:
I think the `EM_TOKEN` is extraneous in the tree. It would be cleaner to initialize to `{}` and then `_.set` directly on the object rather than always in `EM_TOKEN`.

Reply:
Agreed!